gen.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Urgh, this script is not meant to be runnable standalone so adding a shebang feels wrong... oh well

Reply:
https://www.shellcheck.net/wiki/SC2148

It's a practicality thing for the tool so it provide the appropriate tips

Reply:
https://unix.stackexchange.com/questions/424492/how-to-define-a-shell-script-to-be-sourced-not-run

Maybe something like this can be added?

Reply:
Or/and use the other 2 shellcheck ways to define the shell variant?

Reply:
Yeah I think
```suggestion
# shellcheck shell=bash
```
will do the trick then.

Reply:
Changed shebang to the shellcheck directive.